packages/jsii-dotnet-runtime-test/test/Amazon.JSII.Runtime.IntegrationTests/ComplianceTests.cs

```diff
@@ -83,10 +83,10 @@ public void CollectionTypes()
             Assert.Equal("World", types.ArrayProperty[1]);
 
             // map
-            IDictionary<string, double> map = new Dictionary<string, double>();
-            map["Foo"] = 123;
+            IDictionary<string, Number> map = new Dictionary<string, Number>();
+            map["Foo"] = new Number(123);
```

Comment:
How common is "Number" going to be? If users have to start using objects instead of numbers across strongly typed languages, that sucks.

Reply:
`Number` is a type in `jsii-calc-lib`, it's not built in to JSII.

Reply:
Correct. And the fact it is named like so allowed me to discover no fewer than 3 bugs last week, so I think it's a good thing :)